app/services/moderator/merge_user.rb

```diff
@@ -6,7 +6,7 @@ def self.call(admin:, keep_user:, delete_user_id:)
 
     attr_reader :keep_user, :admin, :delete_user_id
 
-    def initialize(admin:, keep_user:, delete_user_id:)
+    def initialize(admin:, keep_user:, delete_user_id:) # rubocop:disable Lint/MissingSuper
```

Comment:
I'm sure there is a good reason, but I don't know why the rule is disable here and in a bunch of other places in the PR.

Reply:
The point of this rule is to respect classic OOP inheritance in the construction of objects. You have a base class with an initializer and then when you inherit and "override" the initializer you, in theory, should delegate the construction of the final object to the parent's initializer, adding the new state that's related only to this class's object. But this is not what we're doing it, we're constructing a whole new initializer, thus the need for the rule disable. See how the base class initializer is different from this one

https://github.com/forem/forem/blob/edd0f2c630e062c1c60094b9fd21837943723e09/app/services/moderator/manage_activity_and_roles.rb#L5-L9